Description: Replace deprecated h_addr with non-deprecated syntax
 This replaces calls to the gethostbyname data structure portion h_addr with
 h_addr_list[0], as several C libs have long-deprecated use of h_addr. See
 https://java.net/jira/browse/SGS_C_CLIENT-12 for a similar example. Worth
 noting too is the following text from the gethostbyname() manpage:
 > ...superseded by getaddrinfo() and getnameinfo()! In particular,
 > gethostbyname() doesn't work well with IPv6...
Forwarded: yes
Author: Rowan Thorpe <rowan@rowanthorpe.com>
Last-Update: 2014-03-03
--- a/src/sflowtool.c
+++ b/src/sflowtool.c
@@ -4116,7 +4116,7 @@
       fprintf(ERROUT, "hostname %s lookup failed\n", hoststr);
       return NO;
     }
-    else tgt->host.s_addr = ((struct in_addr *)(ent->h_addr))->s_addr;
+    else tgt->host.s_addr = ((struct in_addr *)(ent->h_addr_list[0]))->s_addr;
   }
   sscanf(p, "%u", &tgt->port);
   if(tgt->port <= 0 || tgt->port >= 65535) {
@@ -4271,7 +4271,7 @@
 	  fprintf(ERROUT, "netflow collector hostname lookup failed\n");
 	  exit(-8);
         }
-    	sfConfig.netFlowOutputIP.s_addr = ((struct in_addr *)(ent->h_addr))->s_addr;
+    	sfConfig.netFlowOutputIP.s_addr = ((struct in_addr *)(ent->h_addr_list[0]))->s_addr;
 	sfConfig.outputFormat = SFLFMT_NETFLOW;
       }
       break;
